Routing/RedirectRouteProvider.php

```diff
@@ -40,13 +40,16 @@ public function __construct(RedirectRouteRepositoryInterface $redirectRouteRepos
      */
     public function getRouteCollectionForRequest(Request $request)
     {
+        // server encodes the url and symfony does not encode it
```

Comment:
Link to https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/Routing/Matcher/UrlMatcher.php#L91 ?